component-interaction/src/app/user-list/user-list.component.ts

```diff
@@ -66,4 +67,20 @@ export class UserListComponent implements OnInit {
 				contact: this.getRandomText()
 			});
 	}
+
+	public handleSelectedUser(userId: number): void {

```

Comment:
```suggestion
	public handleSelectedUser(selectedUserId: number): void {

```
in order to avoid this.selectedUsersIds.filter((userId) => **userId** !== **userId**);